ground_extract_method: 0    # 0: RANSAC, 1: Himmelsbach
# Himmelsbach Ground Plane Extraction Parameters (For occupancy_grid)
alpha: 0.0873               # Number of segments = (2 * M_PI) / alpha
tolerance: 0.25             # Metric distance from the ground plane to be considered a ground point
Tm: 0.36                    # Slopes greater than this value will be considered non-ground
Tm_small: 0.0873            # Slopes less than this value will be checked for being a plateau
Tb: -1.5                    # Flat regions that are higher than this will be considered non-ground
Trmse: 0.2                  # If the RSME of the line fit exceeds this value, it will be rejected
Tdprev: 0.5                 # Maximum allowed distance between previous line and start of new line
# RANSAC Tuning Parameters
MAX_DISTANCE_RANSAC: 0.10               # Max distance from RANSAC plane fit
MAX_ITERATIONS_RANSAC: 25               # Maximum number of iterations to run RANSAC
GROUND_KALMAN_GAIN: 0.27                # x_hat = x_check + k * (y - x_check)
# Occupancy Grid Parameters
grid_size: 0.1     # Discretization used to create the occupancy grid
neighbor_size: 8    # Options: 4-neighbor or 8-neighbor in connected component clustering
